CHARTER.md

```diff
@@ -10,33 +10,37 @@ The Arcalot Round Table *(ART)* consists of:
+
+
 ## Decision Proposals
 
-Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [art-decisions/proposals](art-decisions/proposals). You can use the included `template.md` file as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
+Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). You can use the included [template file](art-decisions/proposals/template.md) as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a minimum of 4 votes is required with 75% approval majority among the voters.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 1 month after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+Any voting period must last at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal. Proposals can be extended no more than twice and for no longer than 30 days total. Extensions can be made by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension.
```

Comment:
Why "voting period must last" instead of just "Any vote must be open for at least" or something a bit more direct.

Reply:
The reason for a sub-30-day voting period is to implement the proposal in a shorter time-span.
To prevent extension abuse, maybe we should define allowable circumstances to extend a proposal.